postprocess/postgresql.jsonnet

```diff
@@ -0,0 +1,22 @@
+/**
+ * Remove generated postgresql manifests if not required
+ */
```

Comment:
I'm not a huge fan of this, but I don't have a better solution. So I guess we'll leave it..

Reply:
Me neither. See #136 for the database topic. Kapitan should allow render dependencies on a conditional basis like [Helm is doing it](https://helm.sh/docs/chart_best_practices/dependencies/#conditions-and-tags).

Reply:
Created a PR on kapitan to make the helm chart rendering conditional kapicorp/kapitan#842